### DISCUSSION TITLE
Advices regarding MobX-State-Tree and AsyncStorage

### DISCUSSION THREAD

**Discussion by otekdo** [infinitered/ignite#2269] [Q&A] [answered]
First thing to say, huge thanks for this amazing boilerplate! 
I just have 2 questions regarding the way mobx and AsyncStorage is set!

1- Is the ``state-data`` from ``MobX-State-Tree`` being persisted already automatically in ``AsyncStorage``?

2- Does the boilerplate have ``AsyncStorage`` installed with the total storage size limit set to a higher size?
Or do i need to set it higher manually?
https://react-native-async-storage.github.io/async-storage/docs/limits
![image](https://user-images.githubusercontent.com/65221868/196025737-cd0c80eb-9b2c-4f5e-96e7-5eaf98f4d62d.png)

**Comment by frankcalise** [comment, marked as answer]
@otekdo Thanks for the compliments!

1- Sort of, `MobX-State-Tree` state data for navigation is being persisted to AsyncStorage by default.
2- To my knowledge, we have not set the storage size limit to anything different than the default at this time.
